README.md

```diff
@@ -1,3 +1,117 @@
+
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Awesome Books <a name="about-project"></a>
+
+In this project, we built a basic website that allows users to add/remove books from a list. We achieved this by using JavaScript objects and arrays. We also dynamically modified the DOM and added basic events.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- JavaScript
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- A collection that keeps a list of books.
+- A function to add a new book to the collection, with title and author.
+- A function to remove a book from the collection.
+- All books saved in the collection are displayed in the top part of the page.
+- When a user clicks the "Add" button:
+  - A new book is added to the collection.
+  - The new book is displayed in the page.
+- When a user clicks the "Remove" button:
+  - The correct book is removed from the collection.
+  - The correct book dissapears from the page.
+- Data is preserved in the browser's memory by using localStorage.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+Git should be installed on your local machine to clone it using HTTP or SSH. Otherwise you can download the zip file.
+
+You can clone this project by using the command below
+
+Command : git@github.com:IsmailChangezi/awesome-books-3.git
+
+### Setup
+
+Clone this repository to your desired folder
+
+## Author
+
+👤 **Author 1**
+
+- GitHub: [@IsmailChangezi](https://github.com/IsmailChangezi)
+
+👤 **Author 2**
```

Comment:
- [ ] OPTIONAL
Generally, the subheading text _Author1_ should be replaced with your full name in your ReadMe. Kindly remove the text _**Author 2**_, as you're the sole contributor to this project.